modules/administration/pages/troubleshooting/tshoot-container-inspection.adoc

```diff
@@ -0,0 +1,37 @@
+[[troubleshooting-container-inspection]]
+= Troubleshooting Container Image Inspection
+
+
+
+////
+PUT THIS COMMENT AT THE TOP OF TROUBLESHOOTING SECTIONS
+
+Troubleshooting format:
+
+One sentence each:
+Cause: What created the problem?
+Consequence: What does the user see when this happens?
+Fix: What can the user do to fix this problem?
+Result: What happens after the user has completed the fix?
+
+If more detailed instructions are required, put them in a "Resolving" procedure:
+.Procedure: Resolving Widget Wobbles
+. First step
+. Another step
+. Last step
+////
+
+The base container image (BCI) comes with all the software to run {productname}, but—because BCIs are lightweight—they may not come with all tools and libraries you may need for debugging.
```

Comment:
```suggestion
The base container image (BCI) comes with all the software to run {productname}, but because BCIs are lightweight they may not come with all tools and libraries you may need for debugging.
```